### ISSUE TITLE
New comparison items time lag

### ISSUE THREAD

**Issue by Gv26** [Vergo777/RankCompare#8]
The two new items to be compared often appear in the UI at slightly different times (not image load times because this is true for the text beneath them too).
Only update the displayed items once both new items have been selected by the matchmaker.

**Comment by Vergo777** [comment]
I'm going to classify this as one of those things which is "eventually nice to have but we can easily live without for now." Digging into the source of the lag is going to be very time consuming. Maybe the display only once updated approach makes more sense, but again, I don't think this is worth spending too much time on in comparison to other stuff on the table.
